tests/rules/no-multiple-global-stores.test.ts

```diff
@@ -25,48 +25,160 @@ ruleTester().run(path.parse(__filename).name, rule, {
     export class OnePlusExtraOk {
       constructor(private store: Store, data: Service) {}
     }`,
-    `
-    export class FirstOk {
-      constructor(private store: Store, data: Service) {}
-    }
-
-    export class SecondOk {
-      constructor(private store: Store, data: Service) {}
-    }`,
   ],
   invalid: [
-    fromFixture(
-      stripIndent`
-        export class NotOkNoVisibility {
+    {
+      code: stripIndents`
+        export class NotOk1 {
           constructor(store: Store, store2: Store) {}
-                      ~~~~~~~~~~~~                  [${messageId}]
-                                    ~~~~~~~~~~~~~   [${messageId}]
         }`,
-    ),
-    fromFixture(
-      stripIndent`
-        export class NotOkOneVisibility {
-          constructor(store: Store, private store2: Store) {}
-                      ~~~~~~~~~~~~                        [${messageId}]
-                                            ~~~~~~~~~~~~~ [${messageId}]
+      errors: [
+        {
+          column: 13,
+          endColumn: 25,
+          line: 2,
+          messageId: noMultipleGlobalStores,
+          suggestions: [
+            {
+              messageId: noMultipleGlobalStoresSuggest,
```

Comment:
I just noticed that updating `Typescript` and `@typescript-eslint` fixes the problem in SOME cases of the weird `as MessageIds` 🤷‍♂️  (I've removed that typecast where applicable... for the others unfortunately, TS keeps complaining).